### PR TITLE
#DP-1547 feat: added sort by feature

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

File: projects/collection-editor-library/src/lib/components/library-list/library-list.component.html [hunk 1]
```diff
@@ -10,14 +10,12 @@
...
-                          #multiSelect>
-            <sui-select-option>
-            </sui-select-option>
-        </sui-multi-select>
-    </div> -->
+    <div class="sb-w-75">
+      <label class="d-block font-weight-normal fs-0785">{{labelMessages?.lbl?.sortBy}}</label>
+      <div (click)="sortContentList();" libTelemetryInteract
+      [telemetryInteractEdata]="telemetryService.getTelemetryInteractEdata('Sort_By','click','launch', telemetryService?.telemetryPageId, {sortContent: !sortContent})"
+       [ngClass]="{'sb-btn-outline-primary': sortContent}" class="sb-btn sb-btn-outline-gray sb-btn-normal ripple">A - Z</div>
```
Comment (twsvaishali) [inline projects/collection-editor-library/src/lib/components/library-list/library-list.component.html:17]: no resource bundle again? @santoshilimi

File: projects/collection-editor-library/src/lib/components/library-list/library-list.component.spec.data.ts [hunk 2]
```diff
@@ -134,5 +134,13 @@ export const mockData = {
         "resourceType": "Learn",
         "node_id": 29619
     }
+} ,
+   sortContentList: {
```
Comment (twsvaishali) [inline projects/collection-editor-library/src/lib/components/library-list/library-list.component.spec.data.ts:138]: @santoshilimi indent properly

**Review by twsvaishali — APPROVED** [review]

File: projects/collection-editor-library/src/lib/components/library-list/library-list.component.spec.data.ts (hunk 2 above)
Comment (santoshilimi) [reply projects/collection-editor-library/src/lib/components/library-list/library-list.component.spec.data.ts:138]: Done

File: projects/collection-editor-library/src/lib/components/library-list/library-list.component.html (hunk 1 above)
Comment (santoshilimi) [reply projects/collection-editor-library/src/lib/components/library-list/library-list.component.html]: we are not using resource bundles here, using one file to store labels

**Review by venkateshwarans — APPROVED** [review]

**Pull request merged**